pyproject.toml

```diff
@@ -17,6 +17,9 @@ filterwarnings =[
 
   # JAX issues an over-eager warning if os.fork() is called when the JAX module is loaded, even if JAX isn't being used
   'ignore:os\.fork\(\) was called\.:RuntimeWarning',
+
+  # Ignore FutureWarning from pymc.math
+  'ignore:.*(\b(pymc\.math)\b).*:FutureWarning',
```

Comment:
We shouldn't ignore but stop using it.

Reply:
I couldn't find an obvious replacement. The function isn't in `scipy.special`, for example. We could compile the pytensor function just for this test, but it seemed like overkill.

Reply:
Do we need it? If so, we can implement locally with numpy